features/support/sauce_driver.rb

```diff
@@ -0,0 +1,19 @@
+module SauceDriver
+  class << self
+    def config
+      @config ||= YAML.load_file(Rails.root + 'config/saucelabs.yml')
+    end
+
+    def sauce_endpoint
```

Comment:
We could also move this in the settings file and access it with something like `Settings.saucelabs.on_demand_url`.